index.html

```diff
@@ -501,32 +505,124 @@
     callback ActionRequestHandler = any (ActionRequest request);
     callback ObserveRequestHandler = any (ObserveRequest request);
 
+  </pre>
+      <ul>
+        <li>The <dfn>addProperty()</dfn> method xyz.</li>
+        <li>The <dfn>removeProperty()</dfn> method xyz.</li>
+        <li>The <dfn>addAction()</dfn> method xyz.</li>
+        <li>The <dfn>removeAction()</dfn> method xyz.</li>
+        <li>The <dfn>addEvent()</dfn> method xyz.</li>
+        <li>The <dfn>removeEvent()</dfn> method xyz.</li>
+        <li>The <dfn>register()</dfn> method xyz.</li>
+        <li>The <dfn>unregister()</dfn> method xyz.</li>
+        <li>The <dfn>start()</dfn> method xyz.</li>
+        <li>The <dfn>stop()</dfn> method xyz.</li>
+        <li>The <dfn>emitEvent()</dfn> method xyz.</li>
+        <li>The <dfn>onRetrieveProperty()</dfn> method xyz.</li>
+        <li>The <dfn>onUpdateProperty()</dfn> method xyz.</li>
+        <li>The <dfn>onInvokeAction()</dfn> method xyz.</li>
+        <li>The <dfn>onObserve()</dfn> method xyz.</li>
+      </ul>
+      
+      <section data-dfn-for="PropertyRequestHandler">
+        <h2><dfn>PropertyRequestHandler</dfn> callback</h2>
+        <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam.</p>
```

Comment:
I suggest to remove the "Lorem ipsum" stuff, just leave it empty, eventually use "..." or ("TODO") for placeholder, optimally say a short sentence that makes sense for that paragraph.